lib/reporters/base.js

```diff
@@ -33,7 +34,7 @@ exports = module.exports = Base;
  * Enable coloring by default.
  */
 
-exports.useColors = isatty || (process.env.MOCHA_COLORS !== undefined);
+exports.useColors = (isatty && supportsColor) || (process.env.MOCHA_COLORS !== undefined);
```

Comment:
`supportsColor` already checks if stdout is TTY, so I think the `isatty` check could be dropped.